### PR TITLE
Add dataset metadata 

### PR DESCRIPTION
**Pull request by rivettp**
From the Google doc

### CODE REVIEW

**Comment by jgeluk** [comment]
I do not see any reference to the term "Self-describing dataset" (SDD), it would be great if you could add that somewhere, maybe best to do that in the Dataset Metadata section?

There's this page now on the principles website: https://principles.ekgf.org/vocab/sdd/

If you could link to that for "more info" then we can work out all the various details of that metadata there.

File: docs-fragments/pillar/data/capability-area/data-architecture/capability/datasets/background-and-intro.md [hunk 1]
```diff
@@ -1,14 +1,54 @@
 # Background & Intro
 
-!!! warn
+A dataset represents a coherent and distinct data asset, or data product, including unstructured data. 
 
-    Work in progress
+A dataset is a logical view on the EKG for a specific purpose and may not directly represent physical storage or a source of data. It may encompass subsetting (shapes, projection or selection) of a physical data store: (e.g. for ontologies: specific classes and properties; for relational databases specific rows and columns as well as tables).
```
Comment (jgeluk) [inline docs-fragments/pillar/data/capability-area/data-architecture/capability/datasets/background-and-intro.md:5]: Begin by referring to the self-describing principle

File: docs-fragments/pillar/data/capability-area/data-architecture/capability/datasets/background-and-intro.md [hunk 2]
```diff
@@ -1,14 +1,54 @@
...
-# Contribution to the EKG
+Datasets are in general usage outside specific enterprises:
+- governments have public data catalogs e.g. US data.gov for many datasets, including published statistics;
+- datasets cataloged by cloud providers (Google, Amazon) and commercial data publishers (Bloomberg, Factset etc) and may be available by download or API
+- Wikidata
+- scientific datasets submitted with papers to represent experimental data
+
+
+## Dataset Metadata
+The dataset metadata contains information about
```
Comment (jgeluk) [inline docs-fragments/pillar/data/capability-area/data-architecture/capability/datasets/background-and-intro.md:16]: List of options should be synced with https://principles.ekgf.org/vocab/sdd/

**Review by jgeluk — APPROVED** [review]

**Comment by rivettp** [comment]
I wasn't aware of that page. That set of pages seem to be for defining the
terms, it doesn't seem appropriate for that huge amount of content. And the
list is rather aspirational so different aspects are best IMO associated
with different levels of maturity. Do you have an example of a SDD that
covers all those aspects?

**Comment by jgeluk** [comment]
It's a big page yes, many of these sub-topics will have to be moved to
other pages at some point. When we get more content we'll have to.

Most of it is aspirational indeed, similar to the three top levels of the
maturity model. So yes we should rank them per MM-level.

**Pull request merged**